build error with cairo-dock-core / plug-ins 3.6.99

Trying with cairo-dock-core / plug-ins 3.6.99

* https://github.com/Cairo-Dock/cairo-dock-core/commit/d71fa59ce36314b0f87c4856929834a73509b551
* https://github.com/Cairo-Dock/cairo-dock-plug-ins/commit/1dae6258a61ef3d757a53b0f130a78b1ccd0937a

compilation error happends on `Disks` : 

```
FAILED: [code=1] Disks/src/CMakeFiles/cd-disks.dir/applet-init.c.o 
/usr/bin/gcc  -fsanitize=address -fsanitize=undefined -DCAIRO_DOCK_FORCE_ICON_IN_MENUS=1 -DCD_APPLET_MULTI_INSTANCE=\"1\" -DGL_GLEXT_PROTOTYPES=\"1\" -DMY_APPLET_CONF_FILE=\"Disks.conf\" -DMY_APPLET_DOCK_VERSION=\"3.6.99\"
 -DMY_APPLET_GETTEXT_DOMAIN=\"cairo-dock-plugins\" -DMY_APPLET_ICON_FILE=\"icon.png\" -DMY_APPLET_PREVIEW_FILE=\"preview.jpg\" -DMY_APPLET_SHARE_DATA_DIR=\"/usr/share/cairo-dock/plug-ins/Disks\" -DMY_APPLET_USER_DATA_DIR=\
"Disks\" -DMY_APPLET_VERSION=\"0.0.7\" -Dcd_disks_EXPORTS -I/usr/include/cairo-dock -I/usr/include/cairo-dock/gldit -I/usr/include/cairo-dock/implementations -I/usr/include/gtk-3.0 -I/usr/include/cloudproviders -I/usr/incl
ude/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/atk-1.0 -I/usr/lib64/pkgconfig/../../include/dbus-1.0 -I/usr/lib64/pkgconfig/../../lib64/dbus-1.0/include -I/usr/include/gio-unix-2.0 -I/usr/include/librsvg-2.0 
-I/usr/include/gdk-pixbuf-2.0 -I/usr/include/glycin-2 -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/pixman-1 -I/usr/include/harfbuzz -I/usr
/include/freetype2 -I/usr/include/libpng16 -I/usr/include/libxml2 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/sysprof-6 -O2  -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-securit
y -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -march=x86-64 -mtune=generic
 -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -mtls-dialect=gnu2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fno-inline -fno-inline-small-functions -fPIC   -Wall -Werror-implicit-function
-declaration -MD -MT Disks/src/CMakeFiles/cd-disks.dir/applet-init.c.o -MF Disks/src/CMakeFiles/cd-disks.dir/applet-init.c.o.d -o Disks/src/CMakeFiles/cd-disks.dir/applet-init.c.o -c /home/tasaka1/rpmbuild/fedora-specific/
MY-PKG/cairo-dock-plug-ins/nightly/cairo-dock-plug-ins-3.6.99_20260420git1dae625-build/cairo-dock-plugins-3.6.99-20260420git1dae625/Disks/src/applet-init.c
In file included from /home/tasaka1/rpmbuild/fedora-specific/MY-PKG/cairo-dock-plug-ins/nightly/cairo-dock-plug-ins-3.6.99_20260420git1dae625-build/cairo-dock-plugins-3.6.99-20260420git1dae625/Disks/src/applet-init.c:24:
/home/tasaka1/rpmbuild/fedora-specific/MY-PKG/cairo-dock-plug-ins/nightly/cairo-dock-plug-ins-3.6.99_20260420git1dae625-build/cairo-dock-plugins-3.6.99-20260420git1dae625/Disks/src/applet-struct.h:85:9: error: unknown type
 name ‘DBusGProxy’; did you mean ‘GDBusProxy’?
   85 |         DBusGProxy *dbus_proxy_nm;
      |         ^~~~~~~~~~
      |         GDBusProxy
```

Looking at the code, `dbus_proxy_nm` is no longer used.

--- a/Disks/src/applet-config.c
+++ b/Disks/src/applet-config.c
@@ -68,9 +68,6 @@ CD_APPLET_RESET_CONFIG_END
 CD_APPLET_RESET_DATA_BEGIN
 	gldi_task_free (myData.pPeriodicTask);
 	
-	if (myData.dbus_proxy_nm != NULL)
-		g_object_unref (myData.dbus_proxy_nm);
-	
 	CD_APPLET_REMOVE_MY_DATA_RENDERER;
 	
 	g_timer_destroy (myData.pClock);

--- a/Disks/src/applet-struct.h
+++ b/Disks/src/applet-struct.h
@@ -82,7 +82,6 @@ struct _AppletData {
 	
 	// end of shared memory
 	GldiTask *pPeriodicTask;
-	DBusGProxy *dbus_proxy_nm;
 } ;
 
 

